src/util.js

```diff
@@ -429,6 +445,14 @@ function isProgramLikeNode(node) {
   return ["program", "declare", "namespace"].includes(node.kind);
 }
 
+function getNodeKindIncludeLogical(node) {
```

Comment:
I'm not so happy with the name of this function, but after reading what it does it seems hard to come up with a better name. Is the naming from prettier core?

Reply:
@czosel no, `babylon` separate operators on `bin` and `logical`, in php all operators are `bin`

Reply:
Anyway if somebody can find better name - PR welcome. We can change name in any time.

Reply:
Ok... a minor improvement might be `getNodeKindIncludingLogical` and a little comment saying pretty much what you just commented?